catalog/to-consul/resource.go

```diff
@@ -26,6 +27,10 @@ const (
 	// ConsulK8SNS is the key used in the meta to record the namespace
 	// of the service/node registration.
 	ConsulK8SNS = "external-k8s-ns"
+
+	// ConsulK8SHostNameKey is the key used in the meta to track hostname
+	// of the consul-k8s agent syncing.
+	ConsulK8SHostNameKey = "agent-k8s-hostname"
```

Comment:
```suggestion
	SyncCatalogPodNameMeta = "sync-catalog-pod-name"
```

I think better to be explicit about what the value it